chapters/logging.rst

```diff
@@ -0,0 +1,244 @@
+==============================
+Logging configuration in Jina
+==============================
+
+.. meta::
+   :description: A guide on logging configuration in Jina
+   :keywords: Jina, logging
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+-------------------
+
+In order to better understand, monitor, and debug the running and lifetime of Jina's peas, pods, and flow, Jina offers logging to help log the messages.
+
+Before you start
+-------------------
+You have installed the latest stable release of Jina Core according to the instructions found `here <https://docs.jina.ai/chapters/core/setup/index.html>`_ .
+
+
+Overview
+-------------------
+
+Jina logging message at first glance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's take a look at a logging example first. This will help you quickly understand the content of Jina's logging message.
```

Comment:
I would rephrase `Let's take a look at a logging example first. This will help you quickly understand the content of Jina's logging message.` to `Let's start from an logging example`, how do you think?

Reply:
Sure! Will update